### DISCUSSION TITLE
Feature Request: approx_unique does not go higher than count

### DISCUSSION THREAD

**Discussion by Code4SAFrankie** [duckdb/duckdb#12227] [Ideas]
I would like approx_unique to never go higher than count when using summarize, since this would be useful in terms of percentage or cardinality calculations. Can it be capped as such.

I found out that this is an issue when running a summarize on a database primary column key.

**Comment by Tishj** [comment]
I don't think that can be guaranteed, that's just a hunch though
Perhaps some stats could be used to get an approximate count to be used as cap
